src/Controller/SearchController.php

```diff
@@ -44,14 +46,17 @@ public function __invoke(Request $request): Response
             );
         }
 
+        $this->logger->debug("searching for $query");
```

Comment:
I know it's "just" debugging. But maybe something like this format might be safer?

```php
   $logger->debug('User {userId} has logged in', [
        'userId' => $this->getUserId(),
    ]);
```

Reply:
Ps. same remark of course on all the other places like line 55, 59, 76, etc.

Reply:
good call